lib/matplotlib/tests/test_proj_on_rect_patch.py

```diff
@@ -0,0 +1,19 @@
+import math
+import matplotlib.pyplot as plt
+from matplotlib import patches
+import matplotlib
+from matplotlib.testing.decorators import image_comparison
+
+
+@image_comparison(baseline_images=['polar_proj'], extensions=['png'])
+def test_adding_rectangle_patch():
+    fig = plt.figure(figsize=(640/80, 640/80), dpi=80)
```

Comment:
I don't think you need to specify the figsize or dpi, just `plt.figure()` should be fine.